packages/core/tsconfig.json

```diff
@@ -1,9 +1,13 @@
 {
     "extends": "../../tsconfig.json",
     "compilerOptions": {
+        "paths": {
+            "@deriv/*": ["../*/src"]
```

Comment:
[Question]: how about the other paths such as `Components`, `Modules` and so the rest?

Reply:
I forgot to add them. thanks @matin-binary. will add them